web/src/lib/static/loadTasks.ts

```diff
@@ -60,7 +60,7 @@ export const loadTaskByUrlSlug = (urlSlug: string): Task => {
   }
 };
 
-const loadTaskByFileName = (fileName: string, directory: string): Task => {
+export const loadTaskByFileName = (fileName: string, directory: string): Task => {
```

Comment:
So loadTaskByFileName and fetchTaskByFilename are extremely similar and should probably be consolidated into 1 function. However it seems like most of the logic in the fetchTaskByFilename is similar to logic found in loadTasks.ts as a result I think it would be best to have a follow ticket to consolidate all similar logic between the 2 rather than doing some here and some in a follow up ticket. 

https://www.pivotaltracker.com/story/show/185588269